src/main/kotlin/racingcar/domain/RacingCarDistance.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar.domain
+
+import racingcar.contants.ErrorType
+
+class RacingCarDistance(
+    private var distanceValue: Int = 0
+) {
```

Comment:
이렇게 값이 하나뿐인 값 객체는 다음과 같이 값 객체로 최적화가 가능합니다. 
```suggestion
@JvmInline
value class RacingCarDistance(
    private var distanceValue: Int = 0
) {
```
이렇게 표현하면, 컴파일시점에 해당 객체를 일반 필드처럼 인라이닝해서 객체 참조 오버헤드를 줄일 수 있습니다